tools/csv-merger/csv-merger.go

```diff
@@ -594,7 +594,7 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 	ocsCSV.Annotations["features.ocs.openshift.io/enabled"] = `["external"]`
 	// Used by UI to validate user uploaded metdata
 	// Metadata is used to connect to an external cluster
-	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["ceph-rbd"]}`
+	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["ceph-rbd"], "CephCluster": ["monitoring-endpoint"]}`
```

Comment:
```suggestion
	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["ceph-rbd"], "cephCluster": ["monitoring-endpoint"]}`
```

Reply:
@jarrpa The reason for keeping it in caps was : https://github.com/rook/rook/blob/85d72f70b06f7fb480229994cf8c139616aa088a/cluster/examples/kubernetes/ceph/create-external-cluster-resources.py#L470 .

Reply:
Shouldn't matter. All the other entries also have an initial uppercase letter as their official Kind names, yet we still have them lowercase. In fact, is should be plural, so `cephClusters`.

Reply:
Okay sure, I will change it :)

Reply:
Ack, I will update. Thanks

Reply:
done